ui/lib/kv/addon/components/page/secret/overview.hbs

```diff
@@ -0,0 +1,102 @@
+    <li>
+      <LinkTo
+        @route="secret.metadata.index"
+        @models={{array @backend @path}}
+        data-test-secrets-tab="Metadata"
+      >Metadata</LinkTo>
+    </li>
+    <li>
+      <LinkTo @route="secret.paths" @models={{array @backend @path}} data-test-secrets-tab="Paths">Paths</LinkTo>
+    </li>
+    {{#if @canReadMetadata}}
+      <li>
+        <LinkTo
+          @route="secret.metadata.versions"
+          @models={{array @backend @path}}
+          data-test-secrets-tab="Version History"
+        >Version History</LinkTo>
+      </li>
+    {{/if}}
+  </:tabLinks>
+</KvPageHeader>
+
+{{#if (or @metadata @subkeys)}}
+  <div class="flex row-wrap gap-24 has-top-margin-l">
+    <OverviewCard @cardTitle="Current version" @subText={{this.versionSubtext}} class="is-flex-1">
+      <:customTitle>
+        <Hds::Text::Display @weight="semibold" @size="300">
+          Current version
+          {{#if @isDestroyed}}
+            <Hds::Badge @text="Destroyed" @type="outlined" @color="critical" @icon="x" />
+          {{else if @isDeleted}}
+            <Hds::Badge @text="Deleted" @type="inverted" @icon="x" />
+          {{/if}}
+        </Hds::Text::Display>
+      </:customTitle>
+      <:action>
+        {{#if @canUpdateSecret}}
+          <Hds::Link::Standalone
+            @text="Create new"
+            @route="secret.details.edit"
+            @models={{array @backend @path}}
+            @icon="plus"
+            @iconPosition="trailing"
+          />
+        {{/if}}
+      </:action>
+      <:content>
+        <Hds::Text::Display @tag="p" @size="400" @weight="medium" class="has-top-margin-m">
+          {{or @metadata.currentVersion @subkeys.metadata.version}}
+        </Hds::Text::Display>
+      </:content>
+    </OverviewCard>
+
+    {{#if (and (not @isDeleted) (not @isDestroyed))}}
+      {{#let (or @metadata.createdTime @subkeys.metadata.created_time) as |timestamp|}}
+        <OverviewCard
+          @cardTitle="Secret age"
+          @subText="Time since last update at {{date-format timestamp}}."
+          class="is-flex-1"
+        >
+          <:action>
+            {{#if @canReadMetadata}}
+              <Hds::Link::Standalone
+                @text="View metadata"
+                @route="secret.metadata"
+                @models={{array @backend @path}}
+                @icon="arrow-right"
+                @iconPosition="trailing"
+              />
+            {{/if}}
+          </:action>
+          <:content>
+            <Hds::Text::Display @tag="p" @size="400" @weight="medium" class="has-top-margin-m">
+              {{date-from-now timestamp}}
+            </Hds::Text::Display>
+          </:content>
+        </OverviewCard>
+      {{/let}}
+    {{/if}}
+  </div>
+{{/if}}
+
+<Hds::Card::Container @level="mid" @hasBorder={{true}} class="has-padding-l has-top-margin-l">
+  <KvPathsCard @backend={{@backend}} @path={{@path}} @isCondensed={{true}} />
+</Hds::Card::Container>
+
+{{#if @subkeys.subkeys}}
```

Comment:
The subkeys response is 
```
{
  "subkeys": {
    "foo": null,
    "bar": {
      "baz": null
    },
    "quux": null
  },
  "metadata": {
    "created_time": "2021-12-14T20:28:00.773477Z",
    "custom_metadata": null,
    "deletion_time": "",
    "destroyed": false,
    "version": 1
  }
}
```

since there is also metadata attached I opted to pass the whole response instead of separating the args so that we could differentiate where the metadata comes from. (even though it's a little weird to see `@subkeys.subkeys`